### DISCUSSION TITLE
Questions about webpack cache

### DISCUSSION THREAD

**Discussion by wangjieCode** [webpack/webpack#17138] [Q&A]
I have linked an npm package outside of the current project, but after the npm package changes outside of this project, the cache of webpack will still refer to the resources from the first startup within the project. Can I ignore the file cache of certain NPM packages through configuration?

**Comment by alexander-akait** [comment]
You can't changed package path between compilations, they should be immutable

**Reply by wangjieCode** [reply]
Can I ask Webpack not to cache certain NPM packages?

**Reply by alexander-akait** [reply]
You can use this https://webpack.js.org/configuration/cache/#cachemanagedpaths, but I strognly recommend avoid it
